### PR TITLE
doc: add basic Gradle setup using Kotlin

### PR DESCRIPTION
**Pull request by SimonVerhoeven**
## What's changed?

I added a basic setup for those using Gradle + Kotlin

## What's your motivation?

Making the basic setup easier for the readers, especially as Kotlin has become the default for gradle

## Anything in particular you'd like reviewers to focus on?

no

## Have you considered any alternatives or workarounds?

no

### Checklist
- [x] I've added unit tests to cover both positive and negative cases
- [x] I've added the license header to any new files through `./gradlew licenseFormat`
- [x] I've used the IntelliJ auto-formatter on affected files
- [x] I've updated the documentation (if applicable)

### CODE REVIEW

**Comment by timtebeek** [comment]
Thanks a lot for adding this! It's indeed been missing in the docs, but should become more prevalent.

@mike-solomon would you mind going over this? Ties in nicely with the plans we had over on https://github.com/openrewrite/rewrite-recipe-markdown-generator/issues/66

File: running-recipes/getting-started.md [hunk 1]
```diff
@@ -335,6 +337,34 @@ dependencies {
...
 ```
 {% endcode %}
+{% endtab %}
+
+{% tab title="Kotlin" %}
+{% code title="build.gradle.kts" %}
+plugins {
+    `java-library`
+    `maven-publish`
+    id("org.openrewrite.rewrite") version "6.1.19"
```
Comment (timtebeek) [inline running-recipes/getting-started.md:347]: Surprised this line didn't need to be changed in Kotlin, but away from my laptop so can't check right this instant. Calling it out for myself later.

Comment (SimonVerhoeven) [reply running-recipes/getting-started.md:347]: For the groovy setup it's ` id 'org.openrewrite.rewrite' version '6.1.19'`

File: running-recipes/getting-started.md [hunk 2]
```diff
@@ -309,8 +309,10 @@ If you choose to use the `rewrite-recipe-bom`, you won't have to worry about spe
 
 If you choose to not use `rewrite-recipe-bom`, you'll need to specify the version of each rewrite recipe module you use.
 
-Presuming you chose to use the `rewrite-recipe-bom`, your `build.gradle` file should look similar to this:
+Presuming you chose to use the `rewrite-recipe-bom`, your gradle setup should look similar to this:
```
Comment (mike-solomon) [inline running-recipes/getting-started.md:312]: Minor capitalization suggestion:

```suggestion
Presuming you chose to use the `rewrite-recipe-bom`, your Gradle setup should look similar to this:
```

**Review by mike-solomon — APPROVED** [review]
Looks good to me. One minor capitalization fix -- but otherwise looks great. Appreciate you taking the time to add this!

**Pull request merged**